_docs/durability.md

```diff
@@ -4,4 +4,138 @@ title: Durability
 permalink: /docs/durability/
 ---
 
-**Note:** The original durability implementation had a variety of shortcomings and has been replaced by a more robust and performant implementation as discussed at [issue 198](https://github.com/cerner/clara-rules/issues/198).  Documentation on this new durability API is pending.
+### Overview ###
+
+The [clara.rules.durability]({{site.clojuredoc}}clara.rules.durability.html) namespace offers functions serialize and deserialize Clara sessions.  The motivation and goals of this durability layer is described in [issue 198](https://github.com/cerner/clara-rules/issues/198).
```

Comment:
Thanks.  I'll fix it.  I need to do a grammar review over this it looks like.  Hopefully you don't see too many of these.

I also have considered adding perhaps a little more rationale (in a new section added to the bottom) around the decision of the API to be based on streams rather than something a bit more data-oriented (hint: it was all about performance).  I may get to that.
I'm not sure that is that critical though to get the initial docs posted for this feature.